src/firebase-sn.ts

```diff
@@ -1,4 +1,4 @@
-import * as Firebase from 'firebase'
+import {firebase} from './process-firebase'
 import {makeCollections} from './collections'
```

Comment:
Agreed

On Sun, 4 Sep 2016, 17:15 sd, <notifications@github.com> wrote:

> In src/firebase-sn.ts
> <https://github.com/SparksNetwork/sparks-backend/pull/35#discussion_r77446852>
> :
>
> > @@ -1,4 +1,4 @@
> > -import * as Firebase from 'firebase'
> > +import {firebase} from './process-firebase'
> >  import {makeCollections} from './collections'
>
> tbh i think this is an unnecessary abstraction at this point, ive been
> poking around in a branch with this, and i am finding a clean approach of:
>
>    - seneca-firebase-model: doesnt care what "model name" you push it,
>    its just using it as the path for the records youre dealing with, this is
>    talking directly to the firebase reference that it was passed in its options
>    - individual model plugins, eg Profiles
>    - the rest of the app only uses the model plugins
>
> So the only place you're explicitly defining the firebase path of a given
> model is in its plugin. We dont have to worry about crap like writing to
> "Opp" rather than "Opps", because we're sending a message to the Opps
> plugin, and it wont land if we send it to the "Opp" plugin.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SparksNetwork/sparks-backend/pull/35/files/de8d18eb4dd5388ff8258ccb9daeb7fb718fa62b#r77446852>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAfUB-xomOlfE62Kid0aMJzGg0Ifjpoks5qmlPhgaJpZM4J0NRA>
> .
>